src/main/java/com/hedvig/claims/aaafindahomeforme/PaymentRequestRouter.kt

```diff
@@ -0,0 +1,131 @@
+package com.hedvig.claims.aaafindahomeforme
+
+import com.hedvig.claims.commands.AddAutomaticPaymentCommand
+import com.hedvig.claims.commands.AddExpensePaymentCommand
+import com.hedvig.claims.commands.AddIndemnityCostPaymentCommand
+import com.hedvig.claims.commands.AddNoteCommand
+import com.hedvig.claims.commands.AddPaymentCommand
+import com.hedvig.claims.query.ClaimsRepository
+import com.hedvig.claims.serviceIntegration.meerkat.Meerkat
+import com.hedvig.claims.serviceIntegration.meerkat.dto.SanctionStatus
+import com.hedvig.claims.serviceIntegration.memberService.MemberService
+import com.hedvig.claims.web.dto.PaymentType
+import com.hedvig.claims.web.dto.CreatePaymentDto
+import java.time.LocalDateTime
+import java.util.UUID
+import org.axonframework.commandhandling.gateway.CommandGateway
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Component
+
+@Component
+class PaymentRequestRouter(
```

Comment:
To align a bit with how we usually name these things 🤷 
```suggestion
@Service
class ClaimPaymentService(
```